### PR TITLE
Finishing Infosection part

### PR DESCRIPTION
**Pull request by ziadbastawy**

### CODE REVIEW

File: src/components/InfoSection/InfoSection.jsx [hunk 1]
```diff
@@ -1,9 +1,39 @@
 import React from 'react';
+import AvatarImg from './avatar.png';
+import SettingIcon from './setting.js';
+import './InfoSection.css';
 const InfoSection = () => {
-    return <div>
-        InfoSection 
+    // Avatar
```
Comment (majidzeno) [inline src/components/InfoSection/InfoSection.jsx:6]: Can you remove this comment

File: src/components/InfoSection/InfoSection.jsx [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+    // Info
```
Comment (majidzeno) [inline src/components/InfoSection/InfoSection.jsx:7]: And this one too

File: src/components/InfoSection/InfoSection.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  border: 1px solid #ddd;
+  border-radius: .3em;
+  font-weight: 600;
+}
+
+.settingButton {
+  margin-inline-end: 1em;
+}
+
+.settingButton:link, .settingButton:active, .settingButton:visited, .settingButton:hover {
```
Comment (majidzeno) [inline src/components/InfoSection/InfoSection.css:66]: Can you format this to be more readable ?

Comment (ziadbastawy) [reply src/components/InfoSection/InfoSection.css:66]: Done

File: src/components/InfoSection/InfoSection.jsx (hunk 2 above)
Comment (ziadbastawy) [reply src/components/InfoSection/InfoSection.jsx:7]: done

File: src/components/InfoSection/InfoSection.jsx (hunk 1 above)
Comment (ziadbastawy) [reply src/components/InfoSection/InfoSection.jsx:6]: done

**Pull request merged**